src/ViewContext.tsx

```diff
@@ -318,7 +318,7 @@ export function addNodeToPath(
   return [...pathWithRelations, { nodeID, nodeIndex }];
 }
 
-function popPath(viewContext: ViewPath): ViewPath | undefined {
+export function popPath(viewContext: ViewPath): ViewPath | undefined {
```

Comment:
don't export `popPath`, follow what we are doing with the current `DisconnectBtn`